huggingface/tensorflow/inference/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -1,5 +1,127 @@
-# https://github.com/aws/deep-learning-containers/blob/master/available_images.md
-# refer to the above page to pull latest Tensorflow image
+FROM nvidia/cuda:11.0-cudnn8-runtime-ubuntu18.04
 
-# docker image region us-west-2
-FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-inference:2.4.1-gpu-py37-cu110-ubuntu18.04
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
As per the TF 2.4.1 Inference GPU dockerfile, the GPU image is not capable of multi-model inference.
https://github.com/aws/deep-learning-containers/blob/master/tensorflow/inference/docker/2.4/py3/cu110/Dockerfile.gpu#L1-L7
Is there anything on this dockerfile that makes the HF TF 2.4.1 GPU image capable of multi-model inference? If there is, please add a reference to that in the comment above this `LABEL`. If not, please remove this `LABEL` from the dockerfile.